py-polars/polars/expr/expr.py

```diff
@@ -3407,8 +3407,7 @@ def apply(
 
             def wrap_f(x: Series) -> Series:  # pragma: no cover
                 def inner(s: Series) -> Series:  # pragma: no cover
-                    s.rename(x.name, in_place=True)
-                    return function(s)
+                    return function(s.rename(x.name))
```

Comment:
Let's just use `alias` which is 'idomatic'?

```suggestion
                    return function(s.alias(x.name))
```

Reply:
Good point! I completely forgot we had both; definitely only need one, and `alias` is consistent with expressions/etc. Will update the deprecation accordingly.

Reply:
Done... consolidated internal usage on `alias`.